include/picongpu/simulation_defines/param/isaac.param

```diff
@@ -17,27 +17,37 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
+/** @file
+ *
+ * Definition of native fields and density fields of particles which shall be
```

Comment:
It would be great if you describe here in a sentence or two what the `ISAAC` plugin *itself* is doing for a user.

at the end, the description of `@file` is what will show up in the manual (the in-code docs below will later not be rendered anymore in the manual).

Explain the user that ISAAC does *in situ* visualization of fields and what needs to be known simulation-setup wise, such as that ISAAC operates in-memory for native fields, etc.

Concepts and detailed techniques are explained in the manual in the plugins section, but here prepare the user to do the right decisions for the options below, so they do not blow up their memory or do stupid things.

Reply:
Hm, I thought the in-code doc is also be rendered: http://picongpu.readthedocs.io/en/latest/usage/param/core.html#grid-param.

I will add some details ;)